### PR TITLE
Add com.jvieira.tpt.Metronome

### PR DESCRIPTION
**Pull request by jvieiratpt**

### CODE REVIEW

File: com.jvieira.tpt.Metronome.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "master",
+    "sdk": "org.gnome.Sdk",
+    "command": "metronome",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
```
Comment (bilelmoussaoui) [inline com.jvieira.tpt.Metronome.json:12]: Dconf access isn't needed any more.

File: com.jvieira.tpt.Metronome.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
+{
+    "app-id": "com.jvieira.tpt.Metronome",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "master",
```
Comment (bilelmoussaoui) [inline com.jvieira.tpt.Metronome.json:4]:
```suggestion
    "runtime-version": "3.36",
```

File: com.jvieira.tpt.Metronome.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.jvieira.tpt.Metronome.desktop</id>
```
Comment (bilelmoussaoui) [inline com.jvieira.tpt.Metronome.appdata.xml:3]:
```suggestion
  <id>com.jvieira.tpt.Metronome</id>
  <launchable type="desktop-id">com.jvieira.tpt.Metronome.desktop</launchable>
```

Comment (bilelmoussaoui) [reply com.jvieira.tpt.Metronome.appdata.xml:3]: The appdata should be upstreamed if possible

File: com.jvieira.tpt.Metronome.json [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    "command": "metronome",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```
Comment (LeandroStanger) [inline com.jvieira.tpt.Metronome.json:15]:
```suggestion
```

File: com.jvieira.tpt.Metronome.appdata.xml (hunk 3 above)
Comment (LeandroStanger) [reply com.jvieira.tpt.Metronome.appdata.xml:3]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Comment (jvieiratpt) [reply com.jvieira.tpt.Metronome.appdata.xml]: Thank you! I will upstream this changes.

**Comment by LeandroStanger** [comment]
bot, build com.jvieira.tpt.Metronome

**Comment by bilelmoussaoui** [comment]
bot, build com.jvieira.tpt.Metronome

File: com.jvieira.tpt.Metronome.appdata.xml [hunk 5]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bilelmoussaoui) [inline com.jvieira.tpt.Metronome.appdata.xml:1]: It seems that you're having an appdata file upstream already, any reason for including it here as well?

Comment (jvieiratpt) [reply com.jvieira.tpt.Metronome.appdata.xml:1]: No reason, I thought it was necessary...

Comment (bilelmoussaoui) [reply com.jvieira.tpt.Metronome.appdata.xml:1]: It's necessary to be part of the app, Flathub only requires the manifest https://github.com/flathub/flathub/wiki/App-Requirements

Comment (jvieiratpt) [reply com.jvieira.tpt.Metronome.appdata.xml:1]: So, I will make the suggested changes in the upstream appdata file, and I will remove it from here.

File: com.jvieira.tpt.Metronome.json [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dexamples=false",
+                "-Dglade_catalog=disabled",
+                "-Dtests=false"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "branch": "libhandy-0-0",
```
Comment (bilelmoussaoui) [inline com.jvieira.tpt.Metronome.json:37]: Can you add a commit as well to both modules sources?

Comment (jvieiratpt) [reply com.jvieira.tpt.Metronome.json:37]: Yes, the commit and the tag too, right?

Comment (bilelmoussaoui) [reply com.jvieira.tpt.Metronome.json:37]: commit & tag or a commit/branch as the builds should be reproducible

**Comment by bilelmoussaoui** [comment]
bot, build com.jvieira.tpt.Metronome

**Pull request closed without merging**